test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -371,7 +384,11 @@ static address reserve_multiple(int num_stripes, size_t stripe_len) {
     address q = p + (stripe * stripe_len);
     // Commit, alternatingly with or without exec permission,
     //  to prevent kernel from folding these mappings.
+#ifdef __APPLE__
+    const bool executable = exec_supported? (stripe % 2 == 0) : false;
```

Comment:
Nit: please add a space before the `?` operator.

Reply:
Done.